pkg/cloudprovider/providers/azure/azure_client.go

```diff
@@ -168,7 +192,11 @@ func (az *azVirtualMachinesClient) CreateOrUpdate(resourceGroupName string, VMNa
 }
 
 func (az *azVirtualMachinesClient) Get(resourceGroupName string, VMName string, expand compute.InstanceViewTypes) (result compute.VirtualMachine, err error) {
-	az.rateLimiter.Accept()
+	err = createArmRateLimitErr(false, "VMGet")
+	if !az.rateLimiterReader.TryAccept() {
+		return
```

Comment:
Educate me in golang. :)

By implicitly returning (simple `return` without params) both named return types, what are we returning as the value of `result` in the error case?